.vscode/settings.json

```diff
@@ -28,5 +28,6 @@
     "xxhash"
   ],
   "cSpell.ignoreWords": ["yxxx"],
-  "eslint.runtime": "node"
+  "eslint.runtime": "node",
+  "files.trimTrailingWhitespace": false
```

Comment:
why?

Reply:
I have it configured to trim trailing whitespace on save on VS Code (which makes sense for most projects I work on). But in this project from what I can tell trailing whitespace is necessary for tests that test console output (since they make empty lines).

Reply:
I would really like to have all trailing whitespace removed in this project.
Which tests were failing when trimmed?

Reply:
Any test with a snapshot with an empty line. It makes an inline snapshot like this
```
      expect(std.out).toMatchInlineSnapshot(`
        "Uploaded
        test-name
        (TIMINGS)
        Deployed
        test-name
        (TIMINGS)
         
        test-name.test-sub-domain.workers.dev"
      `);
```
Which if you have trim whitespace on save enabled will immediately remove the spaces that are *actually needed for the snapshot*. An alternative solution to this is moving the snapshots to their own files instead and just not editing them manually.

Reply:
Yes, there's significant whitespace on this "empty" line
```
        (TIMINGS)
         
        test-name.test-sub-domain.workers.dev"
```
Remove trailing whitespace would delete all the whitespace on the line, including the significant whitespace required for the test.